tests/drivers/regulator/pmic/src/main.c

```diff
@@ -115,12 +115,11 @@ static void test_basic(void)
 	const struct device *reg_dev, *adc_dev;
 	int rc, adc_reading;
 
-	adc_dev = device_get_binding(DT_LABEL(ADC_NODE));
+	adc_dev = DEVICE_DT_GET(ADC_NODE);
 	reg_dev = device_get_binding(CONFIG_TEST_PMIC_REGULATOR_NAME);
```

Comment:
Wouldn't `CONFIG_TEST_PMIC_REGULATOR_NAME` still use the `label` dts mechanism?

Reply:
if you mean the `zassert_not_null` for `reg_dev`, than that was a mistake in removing.

Reply:
Right, nvm then :)